doc/surveys.md

```diff
@@ -39,89 +41,119 @@ This is used both as the Google Analytics tag to record events against, and used
+Note that for `email` surveys the users email is submitted back to the [feedback](https://github.com/alphagov/feedback) app which actually sends the email. The email address is sent with the `identifier` of the survey and this identifier must match with the `id` of a survey defined in [`app/models/email_survey.rb`](https://github.com/alphagov/feedback/blob/85e07b0c572a91be02b64af1d551df313f2695f9/app/models/email_survey.rb#L24). Make sure you define the survey in both `static` and `feedback`.
 
 ### `url` - required for both `url` surveys and `email` surveys
-Used in a link in the survey that the user is directed to click on. This should be the URL of a smartsurvey -- or other survey page -- that allows the visitor to take the survey. If the url contains the template param `{{currentPath}}` this will be replaced with the current page path.  For example if the `url` param is:
+Used in a link in the survey that the user is directed to click on. This should be the URL of a smartsurvey -- or other survey page -- that allows the visitor to take the survey. If the url contains the template param `{{currentPath}}` this will be replaced with the current page path. For example if the `url` param is:
 
 * `https://www.smartsurvey.com/s/2AAAAAA` - it will be left alone and inserted in the template as-is.
 * `https://www.smartsurvey.com/s/2AAAAAA?c={{currentPath}}` - will be transformed into `https://www.smartsurvey.com/s/2AAAAAA?c=/government/publications/the-kingdom-of-the-crystal-skull` (assuming the page the survey was shown on was https://www.gov.uk/government/publications/the-kindgom-of-the-crystal-skull).
 
-### `template` - OPTIONAL
-This describes the UI that the survey will use to encourage users to sign up.  If omitted the default survey for the `surveyType` will be used.
+### `templateArguments` - OPTIONAL
+This allows you to customise the text in the survey. The available options for customisation are based on the `surveyType`
 
-#### url survey template
-An HTML fragment representing the contents of the survey box. This **MUST** conform to the following minimum structure:
+#### `templateArguments` for `url` survey
+The template for a url survey is as follows:
 
 ```html
 <section id="user-satisfaction-survey" class="visible" aria-hidden="false">
   <div class="survey-wrapper">
     <a class="survey-close-button" href="#user-survey-cancel" aria-labelledby="survey-title user-survey-cancel" id="user-survey-cancel" role="button">Close</a>
-    <h2 class="survey-title" id="survey-title">Heading copy</h2>
+    <h2 class="survey-title" id="survey-title">{{title}}</h2>
     <p>
-      <a class="survey-primary-link" href="javascript:void()" id="take-survey" target="_blank" rel="noopener noreferrer">Link text</a>
-      This will open a short survey on another website
+      <a class="survey-primary-link" href="" id="take-survey" target="_blank" rel="noopener noreferrer">{{surveyCta}}</a>
```

Comment:
I think we're missing the `{{surveyUrl}}` template parameter in the `href` here.